SpbuCsharpHw/BWT.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace  SpbuCsharpHw;
```

Comment:
```suggestion
namespace SpbuCsharpHw;
```
А ещё namespace лучше писать в самом верху файла.